src/components/Homepage-EventsText.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import styles from "../css/Homepage-Events.module.css";
+
+export default props => {
+  return (
+    <div className={styles.text}>
+      <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;600;800&display=swap" rel="stylesheet"></link>
```

Comment:
You shouldn't need this line in any of your files, the font should be applied after my PR gets merged by default